modules/swagger-codegen/src/main/resources/apex/Swagger.cls

```diff
@@ -271,9 +271,7 @@ public class Swagger {
         @TestVisible
         protected virtual void applyAuthentication(List<String> names, Map<String, Object> headers,
                                                    List<Param> query) {
-            for (Authentication auth : getAuthMethods(names)) {
```

Comment:
Hey @muenzpraeger I just ran into an NPE here, in the current version of codegen.
I was planning to maybe fix it (will leave a comment at the relevant line), but I see that you're commenting this out entirely.
Could you provide any insights please?
Because if it turns out you did it for the same reason, then maybe we can adopt here the fix I'm thinking of.

Thanks!

Reply:
I'm no longer actively working here. And move on with the generator to this repo:

https://github.com/openapitools/openapi-generator

Please leave a comment there, if the error still surfaces.

Reply:
Over there, `applyAuthentication()` has been hollowed already, meaning the error (while technically still there) won't surface.
However, I have no idea what that means (there's `// TODO Check auth methods` left in place).
What kind of impact does this have on the generated apex client? Is that method vital in any way?